hybridse/src/codegen/udf_ir_builder_test.cc

```diff
@@ -1007,6 +1007,15 @@ TEST_F(UdfIRBuilderTest, degrees) {
     CheckUdf<double, double>(udf_name, -90.0, -pi/2);
     CheckUdf<Nullable<double>, Nullable<double>>(udf_name, nullptr, nullptr);
 }
+TEST_F(UdfIRBuilderTest, charTest) {
+    auto udf_name = "char";
+    CheckUdf<StringRef, int32_t>(udf_name, StringRef("A"), 65);
+    CheckUdf<StringRef, int32_t>(udf_name, StringRef("B"), 322);
+    CheckUdf<StringRef, int32_t>(udf_name, StringRef("N"), -178);
+    CheckUdf<StringRef, int32_t>(udf_name, StringRef("\0"), 256);
```

Comment:
use `StringRef(1, "\0")`

Reply:
This is the constructor I want, thank you!